plugin/ensime.vim

```diff
@@ -38,6 +37,10 @@ command! -nargs=* -range EnToggleFullType call ensime#com_en_toggle_fulltype([<f
 command! -nargs=* -range EnOrganizeImports call ensime#com_en_organize_imports([<f-args>], '')
 command! -nargs=* -range EnAddImport call ensime#com_en_add_import([<f-args>], '')
 
+function! EnPackageDecl() abort
+  return ensime#com_en_package_decl()
+endfunction
```

Comment:
There's been a misunderstanding, I thought you were talking about the abstraction of `(args, range)` introduced in the wrappers of python as keywords, not about this viml code. In that case, I'm sorry, I completely agree this is something we should remove. Also, I do know this is not something personal. This is programming, let's keep egos away.
  
Are you up for a quick PR to remove those methods and make those renamings? Since you're the one that proposed them, I think it's fair if you carry them out. I haven't read those vim plugins, but I'm sure I've seen this pattern elsewhere, and in popular vim plugins. I do not know if it's a good practice in the community, though.
  
I'm waiting for your PR, cheers!